spanner/spanner_snippets/spanner/spanner_batch_write.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_batch_write_at_least_once]
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/spanner"
+	sppb "cloud.google.com/go/spanner/apiv1/spannerpb"
+	"google.golang.org/grpc/status"
+)
+
+func batchWrite(w io.Writer, db string) error {
+	ctx := context.Background()
+	client, err := spanner.NewClient(ctx, db)
+	if err != nil {
+		return err
+	}
+	defer client.Close()
+
+	singerColumns := []string{"SingerId", "FirstName", "LastName"}
```

Comment:
question: what database schema is used here? What schema is assumed? Maybe explain in a comment.

Reply:
Added a comment.

Reply:
I guess all the samples use schema that is mentioned in start of the public documentation attached below.
https://cloud.google.com/spanner/docs/getting-started/go#create_a_database

Instead of adding the schema here, should we add the link to this schema instead? This makes sample look neat.
@telpirion Can you suggest if this is better?

Reply:
Harsha. This looks neater. Updated. @telpirion - PTAL.

Reply:
Thanks!